plugins/org.eclipse.glsp.graph/src/org/eclipse/glsp/graph/DefaultTypes.java

```diff
@@ -33,21 +33,22 @@ private DefaultTypes() {}
    public static final String COMPARTMENT_HEADER = "comp:header";
    public static final String BUTTON = "button";
    public static final String EXPAND_BUTTON = "button:expand";
-   public static final String ISSUE_MARKER = "marker";
 
    // shapes
    public static final String NODE_CIRCLE = "node:circle";
    public static final String NODE_RECTANGLE = "node:rectangle";
    public static final String NODE_DIAMOND = "node:diamond";
+   public static final String FOREIGN_OBJECT = "foreign-object";
```

Comment:
nit: If we are already touching this file, I'd love to align the structuring with https://github.com/eclipse-glsp/glsp-client/blob/master/packages/protocol/src/model/default-types.ts